### ISSUE TITLE
Freepie IMU apk in google Play

### ISSUE THREAD

**Issue by KyokushinPL** [AndersMalmgren/FreePIE#47]
@AndersMalmgren what do you think about put Freepie IMU apk on google play?

**Comment by AndersMalmgren** [comment]
If someone has a Dev account at google I'm fine with them distributing it there, I dont normaly develop for Android so I do not have an account

**Comment by NixillUmbreon** [comment]
Want me to do it? I've got a dev account.

**Comment by AndersMalmgren** [comment]
Sorry for late reply, missed this some how. Are you still up for it?

**Comment by NixillUmbreon** [comment]
Yeah, I am! What should I put as the name/description/icon? I can take screenshots myself.

**Comment by AndersMalmgren** [comment]
Sorry I have been away this weekend.  

Will the icon in the Android APK source do? https://github.com/AndersMalmgren/FreePIE/blob/master/Lib/Android/FreePIE%20Android%20IMU/res/drawable-xhdpi/ic_launcher.png

Or the icon for FreePIE? 

https://raw.githubusercontent.com/AndersMalmgren/FreePIE/master/FreePIE.GUI/free-pie.ico

How about

> **FreePIE IMU** is a app that enables the user to send IMU data to [FreePIE](http://andersmalmgren.github.io/FreePIE/)

>**FreePIE** (Programmable Input Emulator) is a application for bridging and emulating input devices. It has applications primarily in video gaming but can also be used for VR interaction, remote control, and other applications. A typical application might be controlling the mouse in a PC game using a Wiimote. Device control schemes are customized for specific applications by executing scripts from the FreePIE GUI. The script language is based on the **Python** syntax and offers non-programmers an easy way to interface devices.

>FreePIE is very similar to the popular utility GlovePIE, but encourages open development and integration with any device. The software is designed to allow third party developers to add their own I/O plugins either through direct integration into the core library or through a separately compiled plugin mechanism.

**Comment by NixillUmbreon** [comment]
I'll use the FreePIE icon itself. Thanks for the description.
